atp/client.go

```diff
@@ -465,28 +520,34 @@ func (c *client) prepareResultChannels(
 	return nil
 }
 
-// getResultV2 works with the channels that communicate with the RuntimeMessage loop.
+// getResultV2 communicates with the RuntimeMessage loop to get the .
 func (c *client) getResultV2(
 	stepData schema.Input,
 ) ExecutionResult {
```

Comment:
Does `gofmt` like this?  (It seems like a function with only one parameter ought to fit on one line....)

Reply:
It's fine as long as there is a comment.

Reply:
Sure. Changed. I don't think it's any better or worse in this case.